src/draw.coffee

```diff
@@ -202,20 +207,24 @@ exports.Draw = class Draw
         }
 
       _clearCache: ->
+        @_cacheFlag = true
         if @_cacheFlag
-          minX = minY = Infinity
-          maxX = maxY = 0
-          for point in @_points
-            minX = min minX, point.x
-            maxX = max maxX, point.x
+          if @_points.length is 0
```

Comment:
This was a general bug fix where the bounding box of an empty path would be NaN/NaN instead of 0/0 I discovered when I was making the surrounding dotted-line paths for the floating blocks.